armi/bookkeeping/report/reportingUtils.py

```diff
@@ -1002,3 +1005,85 @@ def makeCoreAndAssemblyMaps(r, cs, generateFullCoreMap=False, showBlockAxMesh=Tr
 
 
 COMPONENT_INFO = "Component Information"
+
+
+def makeParticleFuelDesignReport(r):
+    """Add reports for particle fuel specification and usage
+
+    Parameters
+    ----------
+    r : Reactor
+        Reactor containing particle fuel designs. If none are found, no
+        work is done.
+
+    """
+    if not r.blueprints.particleFuelDesigns:
+        return
```

Comment:
If we add it to the `Composite` class, that would allow clients to check if a specific block or assembly contains particle fuel as well, right? Some possible pathways
- `Component.containsParticleFuel` always returns False as they would be leafs on the composite tree and thus don't contain anything
- Reactor and assembly iterate over children, calling `child.containsParticleFuel` or `any(c.containsParticleFuel() for c in self)`
- Then most of the work happens on the `Block` as a thing that contains components or potentially other blocks?